Fenced_Frames_Ads_Reporting.md

```diff
@@ -127,6 +127,9 @@ registerAdBeacon({
 });
 ```
 
+In this example, the registered URL for the `click` event is 
+`"https://adtech.example.co.jp/click?buyer_event_id=123"`. The registrable domain of this URL is `"example.co.jp"`. [The Privacy Sandbox enrollment attestation model](https://github.com/privacysandbox/attestation#the-privacy-sandbox-enrollment-attestation-model) requires its corresponding site `"https://example.co.jp"` to be enrolled as defined in [site-based enrollment](https://developer.chrome.com/blog/announce-enrollment-privacy-sandbox/#site-based-enrollment). Otherwise the beacon will not be sent when there is a `click` event.
```

Comment:
> The registrable domain of this URL is `"example.co.jp"`

This is unclear.  Is "adtech.example.co.jp" also a registerable domain from that URL?  Can we use a well specified term like origin or eTLD+1? alternately we could clarify by saying "registerable domain (eTLD+1)"?

Reply:
Updated the terms we use.